pkg/collector/corechecks/net/wlan/wlan_test.go

```diff
@@ -456,3 +475,107 @@ func TestWLANNoMetricsWhenWiFiInterfaceInactive(t *testing.T) {
 	mockSender.AssertNumberOfCalls(t, "Gauge", 0)
 	mockSender.AssertNumberOfCalls(t, "Count", 0)
 }
+
+func TestWLANNoiseValid(t *testing.T) {
```

Comment:
nit: could we split into two tests `TestWLANNoiseValidDisabled` and `TestWLANNoiseValidEnabled`?

Reply:
sounds good, thank you